test/index.ts

```diff
@@ -16,27 +16,29 @@
 
 'use strict';
 
-const arrify = require('arrify');
-const assert = require('assert');
-const extend = require('extend');
-const proxyquire = require('proxyquire');
-const through = require('through2');
-const {util} = require('@google-cloud/common-grpc');
+import * as arrify from 'arrify';
+import * as assert from 'assert';
+import * as extend from 'extend';
+import * as proxyquire from 'proxyquire';
+import * as through from 'through2';
+import {util} from '@google-cloud/common-grpc';
 const {v2} = require('../src');
 const PKG = require('../../package.json');
 
+const g: {[index: string]: boolean} = global as {};
```

Comment:
Why is this alias needed? The answer should be included as a comment here. Ideally, a more self-documenting name than `g` can be used.

Reply:
I just switched it back to casts to any during the assignment.  This whole assigning to `global` business is ugly.